src/components/cc-product-card/cc-product-card.js

```diff
@@ -0,0 +1,198 @@
+import { css, html, LitElement } from 'lit';
+import { dispatchCustomEvent } from '../../lib/events.js';
+import { i18n } from '../../lib/i18n.js';
+import '../cc-img/cc-img.js';
+import '../cc-button/cc-button.js';
+import '../cc-badge/cc-badge.js';
+
+/**
+ * @typedef {import('./cc-product-card.types.js').Keyword} Keyword
+ */
+
+/**
+ * A component displaying some information about a product in a card.
+ *
+ * @cssdisplay block
+ *
+ * @event {CustomEvent} cc-product-card:select - Fires the product name when the product is selected.
```

Comment:
nitpick: add detail type:

```suggestion
 * @event {CustomEvent<string>} cc-product-card:select - Fires the product name when the product is selected.
```